src/index.html

```diff
@@ -1,17 +1,182 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Form</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <h1>HTML Form</h1>
+  <script type="text/javascript" src="./main.js"></script>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+    <fieldset>
+      <legend>Personal Information</legend>
+      <label class="form__field">
+        Surname:
+        <input 
+          type="text" 
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          required
+        >
+      </label>
+      <label class="form__field">
+        Name:
+        <input 
+          type="text" 
+          name="name"
+          id="name"
+          autocomplete="off"
+          required
+        >
+      </label>
+      <label class="form__field">
+        How old are You?
+        <input 
+          type="number" 
+          name="age" 
+          id="age"
+          min="1" 
+          max="100" 
+          value="12" 
+          autocomplete="off"
+          required
+        >
+      </label>
+      <label class="form__field">
+        Full date of birth:
+        <input 
+          type="date" 
+          name="dateOfBirth"
+          id="dateOfBirth"
+          required
+        >
+      </label>
+      <label class="form__field">
+        I accept the term of the agreement
+        <input 
+          type="checkbox" 
+          name="conditions" 
+          id="conditions"
+          required
+        >
+      </label>
+    </fieldset>
+    <fieldset>
+      <legend>Registration</legend>
+      <label class="form__field">
+        Email:
+        <input 
+          type="email" 
+          name="email" 
+          id="email" 
+          autocomplete="off" 
+          placeholder="email@example.com"
+          required
+        >
+      </label>
+      <label class="form__field">
```

Comment:
Put empty line between the open & closed tag
```suggestion
    </fieldset>
    
    <fieldset>
      <legend>Registration</legend>
      
      <label class="form__field">
        Email:
        <input 
          type="email" 
          name="email" 
          id="email" 
          autocomplete="off" 
          placeholder="email@example.com"
          required
        >
      </label>
      
      <label class="form__field">
```